### PR TITLE
Pull Request: feat/front-end/mainpage

### PR DESCRIPTION
**Pull request by Kimdoodle**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our convention
- [x] Tests for the changes have been added (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Feature
- [ ] Bugfix
- [ ] Refactor (no functional changes, no api changes)
- [ ] Design changes
- [ ] Comment added
- [ ] Code style update (formatting, local variables)
- [ ] Test code added
- [ ] Chore (other changes)
- [ ] Init
- [ ] Rename
- [ ] Remove
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #5

## What is the new behavior?
메인페이지 기능 추가
- 광고 배너
- 통계 배너
- 소개 배너

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: client/Ad.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function Ad(url) {
+  return (
+    <div className="flex items-center justify-center border-2 border-[#000000]
```
Comment (SangminHann) [inline client/Ad.js]: css와 js는 분리하는게 좋아 보여요

**Comment by SangminHann** [comment]
디렉토리를 분리하면 더 좋을것 같아요

**Comment by JunSeop00** [comment]
LGTM !

File: client/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+
```
Comment (JunSeop00) [inline client/tailwind.config.js:9]: LGTM

**Review by park20011029 — APPROVED** [review]
LGTM

**Review by JunSeop00 — APPROVED** [review]
LGTM

**Review by SangminHann — APPROVED** [review]

**Pull request merged**